trl/trainer/reward_trainer.py

```diff
@@ -94,18 +95,21 @@ def __init__(
                 The optimizer and scheduler to use for training.
             preprocess_logits_for_metrics (`Callable[[torch.Tensor, torch.Tensor], torch.Tensor]`):
                 The function to use to preprocess the logits before computing the metrics.
-            max_length (`int`, defaults to `None`):
```

Comment:
I followed the `transformers` convention to remove deprecated args from public doc strings